packages/wallets/src/features/cashier/modules/Transfer/hooks/useTransferMessages/utils/__tests__/insufficientBalanceMessageFn.spec.ts

```diff
@@ -0,0 +1,85 @@
+import insufficientBalanceMessageFn from '../insufficientBalanceMessageFn';
+
+describe('insufficientBalanceMessageFn', () => {
+    it('should not return the message when the missing accountName', () => {
```

Comment:
```suggestion
    it('should not return the message when accountName is missing', () => {
```

Reply:
Oh, honestly can't believe I wrote that test case sentence